packages/flutter/test/widgets/gesture_detector_semantics_test.dart

```diff
@@ -76,4 +77,484 @@ void main() {
 
     semantics.dispose();
   });
+
+  testWidgets('All registered handlers for the gesture kind are called', (WidgetTester tester) async {
```

Comment:
Also: Can you add a test that shows that replacing just the SemanticsGestureDelegate (e.g. from a delegate with a tap handler to one without) actually updates the semantics tree?

Reply:
Done